src/organisms/cards/SimpleFeaturedPolicyCard/index.js

```diff
@@ -33,53 +33,58 @@ function SimpleFeaturedPolicyCard(props) {
   return (
     <div className={classnames(...classes)}>
       <div className={styles['content']}>
-        { premium.price ?
-          <div className={styles['premium']}>
-            <Text
-              type={3}
-              font='a'
-            >
-              {formattedPremium} <Text tag='span' type={11} font='a' spaced color='neutral-2'>{`/${premium.format.toUpperCase()}`}</Text>
-            </Text>
-
-            {premium.tooltip}
-          </div>
-          :
-          <Text type={7} font='a'>{premium.defaultText}</Text>
-        }
+        { premium.price
+          ? (
+            <div className={styles['premium']}>
+              <Text
+                type={3}
+                font='a'
+              >
+                {formattedPremium}
+                {' '}
+                <Text tag='span' type={11} font='a' spaced color='neutral-2'>{`/${premium.format.toUpperCase()}`}</Text>
+              </Text>
+
+              {premium.tooltip}
+            </div>
+          )
+          : <Text type={7} font='a'>{premium.defaultText}</Text>}
 
         <Spacer size={24} />
 
         {
-          carrierLogo.type !== 'img' ?
-            <div className={styles['carrier-info']}>
-              { carrierLogo }
-            </div>
-            :
-            <div className={styles['carrier-logo']}>
-              { carrierLogo }
-            </div>
+          carrierLogo.type !== 'img'
+            ? (
+              <div className={styles['carrier-info']}>
+                { carrierLogo }
+              </div>
+            )
+            : (
+              <div className={styles['carrier-logo']}>
+                { carrierLogo }
+              </div>
+            )
         }
 
-
         {
           onContinue &&
-            <React.Fragment>
-              <Spacer size={36} />
+            [
```

Comment:
what rule removed `React.Fragment`?

Reply:
I think this was a linter change? There's no error thrown when I add it back in so there's no reason I would have removed it.